armotypes/vulnerabilitytypes.go

```diff
@@ -9,11 +21,175 @@ type VulnerabilityJobParams struct {
 	JobID           string `json:"jobID,omitempty"`
 }
 
+type VulnerabilityWorkload struct {
+	Id               string                     `json:"id"`
+	Name             string                     `json:"name"`
+	Namespace        string                     `json:"namespace"`
+	Type             string                     `json:"type"`
+	ClusterName      string                     `json:"clusterName"`
+	ClusterShortName string                     `json:"clusterShortName"`
+	CustomerGUID     string                     `json:"customerGUID"`
+	ImagesCount      int                        `json:"imagesCount"`
+	SeverityStats    map[string]SeverityDetails `json:"severityStats"`
+	RiskFactors      []RiskFactor               `json:"riskFactors"`
+	Labels           []string                   `json:"labels"`
+}
+
+type SeverityDetails struct {
+	Total   int             `json:"total"`
+	TopCves []TopCveDetails `json:"topCves"`
+}
+
+type TopCveDetails struct {
+	Name string `json:"name"`
+}
+
+type VulnerabilitiesImage struct {
```

Comment:
This should be container

Reply:
we show images in UI not containers

Reply:
But an image does not have a container and cluster 
a container has an image and cluster